sdk/go/common/workspace/plugins.go

```diff
@@ -1324,12 +1430,17 @@ func (d *pluginDownloader) downloadToFileWithRetry(pkgPlugin PluginSpec) (string
 	return path.(string), nil
 }
 
-// DownloadToFile downloads the given PluginSpec to a temporary file
-// and returns that temporary file.
-//
-// This has some retry logic to re-attempt the download if it errors for any reason.
-func (d *pluginDownloader) DownloadToFile(pkgPlugin PluginSpec) (*os.File, error) {
-	tarball, err := d.downloadToFileWithRetry(pkgPlugin)
+// DownloadToFile downloads the given PluginSpec to a temporary file and returns that temporary file. This has
+// some retry logic to re-attempt the download if it errors for any reason.
+func (d *pluginDownloader) DownloadToFile(ctx context.Context, pkgPlugin PluginSpec) (*os.File, error) {
+	return d.DownloadToFile(context.Background(), pkgPlugin)
```

Comment:
Context snuck in here, I think this is what makes it fail to compile, and makes the main test suite fail to run (because go can't figure out the list of tests)

```suggestion
func (d *pluginDownloader) DownloadToFile(pkgPlugin PluginSpec) (*os.File, error) {
	return d.DownloadToFileWithContext(context.Background(), pkgPlugin)
```